test/t/unit/test_unit_command_offset.py

```diff
@@ -51,6 +56,7 @@ def test_1(self, bash, functions):
             ("cmd3", wordlist),
             ("cmd4", []),
             ("cmd5", ["0"]),
+            ("cmd6", wordlist),
```

Comment:
Could you separate the test from `test_2`? As in the doc comment, `test_2` is the test template for the completion retry requested by the exit status 124.

Also, the test seems to fail in my environment currently. I wondered how this could get passed [CI test #601](https://github.com/scop/bash-completion/actions/runs/2968137118), but I noticed that actually the test `test_unit_command_offset.py` isn't run in the CI test because it is not included in the [EXTRA_DIST](https://github.com/scop/bash-completion/blob/40aa4dc3c75791f43f850871466039768b3048ad/test/t/Makefile.am#L3)...

Reply:
Separated, and tests enhanced to actually work.